src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Link.js

```diff
@@ -34,9 +38,52 @@ export default class Link extends React.Component<FieldTypeProps<LinkValue>> {
             },
         } = this.props;
 
+        const enabledAttributes = [];
+
+        if (enableAttributes !== undefined) {
+            if (!isArrayLike(enableAttributes)) {
```

Comment:
Sorry miscommunicated. The `enable_attributes` is a `boolean` flag which just enables every attribute for the content manager. There is not longer a limitation here for specific attributes or not.